src/demo/CistromeExplorer.js

```diff
@@ -325,7 +83,14 @@ export default function CistromeExplorer() {
     }
 
     return (
-        <div className="cistrome-explorer">
+        <div className="cistrome-explorer"
+            onKeyDown={e => {
+                if(
+                    (e.key === 'Esc' || e.key === 'Escape') && isSettingVisible
+                ) {
+                    setIsSettingVisible(false);
+                }
+            }}>
```

Comment:
It seems like the Escape key only works after I have clicked the `<select/>` dropdown element. Would you be able to move this to an event listener on the window?

```js
useEffect(() => {
    function closeSidebar(e) {
        if((e.key === 'Esc' || e.key === 'Escape') && isSettingVisible) {
            setIsSettingVisible(false);
        }
    }
    window.addEventListener("keydown", closeSidebar);
    return () => window.removeEventListener("keydown", closeSidebar);
}, [isSettingVisible]);
```

Reply:
Thanks for suggesting this! This is much consistent. Included the Toolkit view here as well.